coderedcms/project_template/basic/website/migrations/0001_initial.py

```diff
@@ -5,7 +5,7 @@
  """
 {% endcomment %}
 {% verbatim %}
-# Generated by Django 2.2.4 on 2019-08-03 16:14
+# Generated by Django 2.2.4 on 2019-08-12 14:12
```

Comment:
Also need to update the sass template at project_template/sass/

Reply:
I meant that the whole `0001_initial.py` migration needs to also be copied to sass template. basic and sass should have the same migration. Not just the dates updated (I just commented on that line to make a comment).